packages/askui-nodejs/src/lib/interactive_cli/create-example-project.ts

```diff
@@ -23,28 +26,31 @@ export class CreateExampleProject {
   constructor(readonly cliOptions: CliOptions) {
     this.baseDirPath = process.cwd();
 
-    this.exampleFolderName = this.getExampleFolderName();
-    this.targetExampleFolderPath = path.join(
+    this.targetDirectoryName = this.getTargetFolderName();
+    this.targetDirectoryPath = path.join(
       this.baseDirPath,
-      this.exampleFolderName,
+      this.targetDirectoryName,
     );
 
     this.askUIControllerUrl = 'https://docs.askui.com/docs/general/Components/AskUI-Controller';
     this.helperTemplateConfig = {};
   }
 
-  private getExampleFolderName(): string {
-    if (this.cliOptions.projectName) {
-      return this.cliOptions.projectName.trim();
+  private getTargetFolderName(): string {
+    if (this.cliOptions.askuiWorkflows) {
+      if (/\s/g.test(this.cliOptions.askuiWorkflows.trim())) {
+        throw new TargetDirectoryError('Directory name must not contain whitespaces');
```

Comment:
The user is going to be the consumer of the error message. I think, currently, it would not be clear where she/he/... went wrong (`Directory name` could be a lot of things from my perspective...).

Maybe, something like
```suggestion
        throw new TargetDirectoryError('AskUI Workflows (-aw, --askui-workflows) directory name must not contain whitespaces');
```